module/power_domain/src/mod_power_domain.c

```diff
@@ -711,9 +711,9 @@ static int initiate_power_state_transition(struct pd_ctx *pd)
     status = pd->driver_api->set_state(pd->driver_id, state);
 
     mod_pd_ctx.log_api->log(MOD_LOG_GROUP_DEBUG,
-        "[PD] %s: %s->%s, %e\n", fwk_module_get_name(pd->id),
+        "[PD] %s: %s->%s, %s (%d)\n", fwk_module_get_name(pd->id),
         get_state_name(pd, pd->state_requested_to_driver),
-        get_state_name(pd, state), status);
+        get_state_name(pd, state), fwk_err2str(status), status);
```

Comment:
Lots of instances of the old name still here:

`fwk_err2str(status)` -> `fwk_status_str(status)`

Reply:
this is want happen when pushing without testing :(
my bad... i'll fix.

Reply:
We've all done it!